src/bin/dolos/common.rs

```diff
@@ -62,6 +69,18 @@ pub fn open_data_stores(config: &crate::Config) -> Result<Stores, Error> {
 pub fn setup_tracing(config: &LoggingConfig) -> miette::Result<()> {
     let level = config.max_level;
 
+    let exporter = LogExporter::builder()
+        .with_http()
+        .with_endpoint("http://localhost:4318/v1/logs")
+        .with_protocol(Protocol::HttpJson)
+        .build().unwrap();
+
+    let provider = LoggerProvider::builder()
+        .with_simple_exporter(exporter)
```

Comment:
we want to use `with_simple_exporter` since it's a lot more responsive in dev mode (and for Paima you're running everything on the same machine so the network overhead is minimal), but for production where you have different machines `with_batch_exporter` will work a lot better